### DISCUSSION TITLE
Add Manage EDI/API permissions option to Seller Central - Seller & Vendor Auth parity please!

### DISCUSSION THREAD

**Discussion by dashapplications** [amzn/selling-partner-api-models#621] [Ideas] [answered]
Currently, a vendor can assign an 'Admin' or 'Contributor' role to a given user invited to the account. The 'Admin' role can grant an application access for the purposes of SP-API developer authentication. So can a 'Contributor' with "Manager EDI/API" permission granted.

However, there is no such thing on the Seller side of authentication. As was true with the MWS API, only the Admin of the account (the single Admin user login) can authorize a third party application as a developer to the account. This has to be done manually with only marginal improvements to the workflow in the SP-API. The biggest issue with this is that an Agency or Aggregator has to ask the Admin of the seller account to authorize every single application or tool the Agency/Aggregator would like to use to help manage the account. Imagine doing that across 50+ accounts - reaching out to the Admin, having them authorize every single application with the dev ID and then having to pass the Auth Token value back to the Agency who then passes it to the developer - it's an extremely inefficient process for everyone involved and obviously unnecessary. Even with the improvement to allow the Admin to use Login with Amazon for a given app, it still requires the Admin to authorize every single application and management of the auth grant on the Agency/Aggregator's part.

A simple fix to this would be to add the same Admin and/or Contributor role to Seller Central with the "Manage EDI/API permissions" permission option. Then the Admin of a seller central account only has to give the Agency/Aggregator the "Manage EDI/API permissions" permission, after which the Agency/Aggregator can authorize the tools and applications they require to effectively manage the account.

**Comment by krithic15** [comment, marked as answer]
Hello @dashapplications ,
Thank you for your suggestion. We have submitted your feature request to the appropriate team.
Thanks,
Krithi, Solutions Architect
Selling Partner Developer Services
